src/network/matrix/matrix.ts

```diff
@@ -0,0 +1,175 @@
+/// <reference path="./matrix.type.d.ts" />
+import sdk, { MatrixClient } from 'matrix-js-sdk'
+import mitt from 'mitt'
+export const endpoint = 'https://matrix.vampire.rip'
```

Comment:
better not use this...

Reply:
We'll change to other url when go to production